bin/peas_shell

```diff
@@ -270,7 +292,7 @@ class PanSensorShell(cmd.Cmd):
 
     def do_change_delay(self, *arg):
         """Change the timing between reads from the named sensor."""
-        # NOTE: For at least the Arduinos, we should not need a delay and a timer, but 
```

Comment:
I'll need to change this command later to match the changes I'm making for reading from the Arduinos.